src/charts/tooltip.js

```diff
@@ -236,9 +246,35 @@ export default function Tooltip (_container, isLegend = false) {
 
   function drawTitle () {
     let title = config.tooltipTitle || cache.title
+    // translate bin from human readable code to d3 time format specifier
+    // TO DO: handle special cases such as "decade" that should display an non-normal value like "1991 - 2000"
+    const binTranslation = {
+      "1c": "",
+      "10y": "",
+      "1y": "%Y",
+      "1q": "",
+      "1mo": "%B",
+      "1s": "%S",
+      "1m": "%M",
+      "1h": "%H",
+      "1d": "%A",
+      "1w": "%U"
+    }
 
-    if (typeof title === "object") {
-      title = d3.utcFormat(config.dateFormat)(title)
+    // format date if we have a date
+    if (title instanceof Date) {
+      const { binningResolution } = config;
+      let specifier = binTranslation[binningResolution]
+
+      if (specifier) {
+        title = d3.utcFormat(specifier)(title)
+      } else if (binningResolution === "1q") {
+        // handle bin translation for bin types not available in d3-time (century, decade, quarter)
+        // currently only handling "1q"
+        title = formatOddDateBin(binningResolution, title, data)
```

Comment:
Why do you need data here if you don't use it in formatOddDateBin?

Reply:
my bad, I thought I had removed that last arg